kingpin/actors/aws/cloudformation.py

```diff
@@ -563,6 +551,8 @@ def _create_stack(self, stack):
 
         if self.option("role_arn"):
             cfg["RoleARN"] = self.option("role_arn")
+        elif KINGPIN_CFN_DEFAULT_ROLE_ARN:
+            cfg["RoleARN"] = KINGPIN_CFN_DEFAULT_ROLE_ARN
```

Comment:
This is the only "real" change in this PR. There is one other place we do this setting and there is an entry in the aws settings file to support this as an env var.